### PR TITLE
add text align on calculator component

### PR DESCRIPTION
**Pull request by WesleyFaveri**

### CODE REVIEW

File: src/interface.ts [hunk 1]
```diff
@@ -98,6 +98,12 @@ export interface CalculatorCommonProps {
    * Text change event.
    */
   onTextChange?: (text: string) => void
+
+
+  /**
```
Comment (budiadiono) [inline src/interface.ts:103]: This comment causing `[tslint] asterisks in jsdoc must be aligned [jsdoc-format]` error raised. Please do :
```
/**
   * Digit align display
   */
```
instead and remove extra line above doc comment.

File: src/interface.ts [hunk 2]
```diff
@@ -98,6 +98,12 @@ export interface CalculatorCommonProps {
...
+  * Digit align display
+  */
+  displayTextAlign?: string;
```
Comment (budiadiono) [inline src/interface.ts]: Instead of using `string` as type, please use `"auto" | "left" | "right" | "center" | "justify"` instead for type safety.

**Review by budiadiono — CHANGES_REQUESTED** [review]
This pull request looks good! Just need some small changes and also you need to update the test snapshot by doing `yarn test -u`.

**Comment by WesleyFaveri** [comment]
Done, i think

**Comment by budiadiono** [comment]
Travis still rejects it, I think you need to update the test snapshots by doing `yarn test -u` and commit the snapshots changes.

**Pull request merged**